### PR TITLE
[11.0][MIG] account_mass_reconcile_transaction_ref

### PR DESCRIPTION
**Pull request by grindtildeath**
Based on :
 - [x] #222

### CODE REVIEW

File: account_mass_reconcile_transaction_ref/__manifest__.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
 # © 2013-2018 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 {
     'name': 'Mass Reconcile Transaction Ref',
-    'version': '10.0.1.1.0',
+    'version': '11.0.1.1.0',
```
Comment (simahawk) [inline account_mass_reconcile_transaction_ref/__manifest__.py]: not sure what's the policy for this repo or module but this should be `11.0.1.0.0`.
Personally, I'm not against keeping the same version tho ;)

File: account_mass_reconcile_transaction_ref/models/__init__.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # © 2013-2018 Camptocamp SA
```
Comment (simahawk) [inline account_mass_reconcile_transaction_ref/models/__init__.py:1]: pls replace w/ "Copyright"

File: account_mass_reconcile_transaction_ref/tests/test_account_reconcile_transaction_ref.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+             cls.env.ref('account.data_account_type_receivable').id)
+        ], limit=1)
+        account_revenue = cls.env['account.account'].search([
+            ('user_type_id', '=',
+             cls.env.ref('account.data_account_type_revenue').id)
+        ], limit=1)
+        sales_journal = cls.env['account.journal'].search([
+            ('type', '=', 'sale')], limit=1)
+        # Create invoice
+        cls.cust_invoice = cls.env['account.invoice'].create({
```
Comment (simahawk) [inline account_mass_reconcile_transaction_ref/tests/test_account_reconcile_transaction_ref.py:27]: pls, disable tracking on the whole env

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by grindtildeath** [comment]
@simahawk done :wink:

**Review by simahawk — APPROVED** [review]

**Review by tschanzt — APPROVED** [review]

**Pull request merged**